### ISSUE TITLE
"dirty" in uname for msys2-runtime v3.4.3 ??

### ISSUE THREAD

**Issue by yashpalgoyal1304** [msys2/MSYS2-packages#3486]
* why does the `uname -a` shows "dirty" in "3.4.3-dirty.x86_64 2022-12-19 20:20 UTC" for recent version of `msys2-runtime-3.4.3-4` ??

* and where's the changelog for that? 
    * i searched at the upstream URL: https://www.cygwin.com/ but couldn't find it
    * i also ran the following command but returned empty handed for all the packages i tried it for: `nano,` `mintty,` `less,` `info,` `make,` `python`

```console
$ pacman -Qc msys2-runtime
error: no changelog available for 'msys2-runtime'.
```

_Originally posted by @yashpalgoyal1304 in https://github.com/msys2/MSYS2-packages/issues/3012#issuecomment-1365274097_

**Comment by jeremyd2019** [comment]
Should be fixed in #3485 (yep, shouldn't have transferred it)

**Comment by yashpalgoyal1304** [comment]
> yep, shouldn't have transferred it

😆 yeah, it's sometimes a bit confusing which repo an issue belongs to. glad to have been replicated 🤣
